README.md

```diff
@@ -0,0 +1,55 @@
+# testcafe-react-selectors
+
+An extension which simplifies the work with ReactJS components for [Tescafe](https://github.com/DevExpress/testcafe) e2e testing framework
+
+##Install
+
+`$ npm install testcafe-react-selectors`
+
+##Usage
+
+####Create selector for ReactJS component
+`ReactSelector` allows create testcafe `Selector` being passed name of component class or nested component element.
+E.g. `ReactSelector('TodoList')`, `ReactSelector('TodoList TodoItem')`, `ReactSelector('TodoItem span')`  etc.
+You are free combine this selectors with testcafe `Selector` filter function like `.withText`, `.nth` and [others](http://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html#functional-style-selectors)
+
+```js

```

Comment:
Did you mean find nested component as ReactSelector('Component').find('NestedComponent')?
No, it's impossible.